wire-ios-sync-engine/Tests/Source/Synchronization/Transcoders/ZMSelfTranscoderTests.m

```diff
@@ -75,7 +75,6 @@ - (void)tearDown
     
     ZMSelfStrategyPendingValidationRequestInterval = self.originalRequestInterval;
     
-    [self.realClientRegistrationStatus tearDown];
```

Comment:
question: did we remove the tearDown method?

Reply:
Yes, it was unnecessary

Reply:
Was removed when I converted ZMClientRegistrationStatus to Swift.